src/date.js

```diff
@@ -63,6 +63,7 @@ Globalize.prototype.dateFormatter = function( options ) {
 	cldr.on( "get", validateRequiredCldr );
 	pattern = dateExpandPattern( options, cldr );
 	properties = dateFormatProperties( pattern, cldr );
+	properties.calendar = Globalize.calendars[cldr.attributes.calendar];
```

Comment:
Inside `dateFormatProperties`, you won't have a reference for `Globalize` anymore. Instead of using `Globalize.calendars` as a hub for calendars, simply use something independent, e.g., `calendar/calendars.js`.


```js
define([
	"./calendars/Gregorian-date",
	"./calendars/Hebrew-date",
	"./calendars/Islamic-date"
], function(calendarsGregorianDate, calendarsHebrewDate, calendarsIslamicDate) {

return {
  gregorian: calendarsGregorianDate,
  hebrew: calendarsHebrewDate,
  islamic: calendarsIslamicDate
};

});
```